src/guides/v2.3/howdoi/admin/customize_admin.md

```diff
@@ -0,0 +1,233 @@
+---
+group: how-do-i
+subgroup: admin-customize-page
+title: Customize Magento Admin Design
```

Comment:
```suggestion
title: Customize Magento Admin design
```

Reply:
I'd have preferred an all capital title but I'll go with it :)